docs/index.asciidoc

```diff
@@ -306,7 +306,7 @@ However, if one pattern depends on a field created by a previous pattern, separa
       }
       grok {
         match => {
-          "rest => "a number %{NUMBER"number}, and a word %{WORD:word}"
+          "rest" => "a number %{NUMBER"number}, and a word %{WORD:word}"
```

Comment:
This is a good opportunity to also fix the use of the NUMBER expression, where the filter name needs to be separated with a colon:

```suggestion
          "rest" => "a number %{NUMBER:number}, and a word %{WORD:word}"
```

Reply:
There's already an approved PR from _months_ ago that fixes both of these issues, but nothing is getting merged in this repo at the moment, unfortunately. 😐 Are you able to poke someone? 😇